apps/public-web/src/EShopOnAbp.PublicWeb/EShopOnAbpPublicWebModule.cs

```diff
@@ -93,6 +94,8 @@ public override void ConfigureServices(ServiceConfigurationContext context)
             );
         });
 
+        Configure<RemoteServices>(options => { options.Default.BaseUrl = configuration["RemoteServices:Default:BaseUrl"]; });
```

Comment:
We don't need to configure new option, we already have it: `AbpRemoteServiceOptions`